doc/cug.tex

```diff
@@ -3558,6 +3558,50 @@ \subsubsection{Trigger Types}
                 """
 \end{lstlisting}
 
+\paragraph{Writing Efficient Inter-Family Triggering}
+\label{EfficientInterFamilyTriggering}
+
+While cylc allows writing dependencies between two families such as this:
+
+\begin{lstlisting}
+[scheduling]
+    [[dependencies]]
+        graph = """
+    FAM1:start-any => FAM2
+                """
+\end{lstlisting}
+
+it is important to consider the number of dependencies this will generate. In
+the above example, each member of \lstinline=FAM2= has dependencies pointing at
```

Comment:
Maybe *Cylc allows writing ... blah ... but it is important...*?   With "while" the first line looks a bit like an unfinished sentence because of the split paragraph.